sw/device/silicon_creator/lib/sigverify/BUILD

```diff
@@ -0,0 +1,220 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules:opentitan.bzl", "OPENTITAN_CPU")
+load("//rules:cross_platform.bzl", "dual_cc_device_library_of", "dual_cc_library", "dual_inputs")
+load(
+    "//rules:opentitan_test.bzl",
+    "opentitan_functest",
+    "verilator_params",
+)
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "rsa_key",
+    hdrs = ["rsa_key.h"],
+)
+
+cc_library(
+    name = "testvectors",
+    hdrs = ["sigverify_tests/sigverify_testvectors.h"],
+)
+
+dual_cc_library(
+    name = "mod_exp_ibex",
+    srcs = dual_inputs(
+        device = ["mod_exp_ibex.c"],
+        host = ["mod_exp_ibex_mock.cc"],
+    ),
+    hdrs = dual_inputs(
+        host = ["mod_exp_ibex_mock.h"],
+        shared = ["mod_exp_ibex.h"],
+    ),
+    deps = dual_inputs(
+        device = [
+            "//sw/device/lib/base:macros",
+            "//sw/device/lib/base:memory",
+        ],
+        host = [
+            "//sw/device/lib/base/testing:global_mock",
+            "//sw/device/silicon_creator/testing:mask_rom_test",
+            "@googletest//:gtest",
+        ],
+        shared = [
+            ":rsa_key",
+            "//sw/device/silicon_creator/lib:error",
+        ],
+    ),
+)
+
+cc_test(
+    name = "mod_exp_ibex_unittest",
+    srcs = ["mod_exp_ibex_unittest.cc"],
+    deps = [
+        dual_cc_device_library_of(":mod_exp_ibex"),
+        "@googletest//:gtest_main",
+    ],
+)
+
+opentitan_functest(
+    name = "mod_exp_ibex_functest",
+    srcs = ["mod_exp_ibex_functest.c"],
+    verilator = verilator_params(
+        timeout = "long",
+        tags = [
+            "manual",
```

Comment:
Can you put a todo for the manual tests linking https://github.com/lowRISC/opentitan/issues/11860 IDK if we'll get to it but it's nice to know why it's different.